QADI_TOOLS_README.md

```
@@ -0,0 +1,286 @@
+```bash
+uv run python compare_qadi_approaches.py "how to reduce stress"
+```
+
+5. **See genetic evolution in action**:
+```bash
+uv run python qadi_evolution_demo.py "how to reduce stress"
+```
+
+## 🎯 When to Use Which Tool
+
+### Use `qadi.py` when:
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)